sherpa/astro/xspec/__init__.py

```diff
@@ -1752,6 +1753,50 @@ def mkabund(name: str,
                        hard_min=minval, hard_max=maxval, frozen=True)
 
 
+
+def eval_xspec_with_fixed_norm(func: Callable) -> Callable:
+    """Decorator to evaluate an `XSAdditativeModel` with norm=1.
+
+    The norm is then applied in Python, instead of in the XSPEC code.
+    This is a speed optimization. For a single call to an XSPEC model, there
+    is no advantage, because the XSPEC model needs to be called anyway; it is
+    just called with norm=1 instead of the actal norm and the output of the
```

Comment:
```suggestion
    just called with norm=1 instead of the actual norm and the output of the
```

Although to be honest I think a lot of this can be removed; I just haven't bothered to come up with a suggestion.

Reply:
It's part developer documentation to explain *why* we do this. I'm thinking of another person reading this a few years from now when the CXC is gone and none of the current developers are around to ask questions or a user who studies how we implemented it, maybe even because they want to do something similar in their code wrapping XSPEC in Julia or  whatever the next fancy language after Python will bei.  I can remove it, but a few lines of comment also don't cost us much.